script/playercomponent.js

```diff
@@ -214,7 +216,8 @@ define(
         var hasNextCDN = mediaMetaData.urls.length > 1;
         var aboutToEndVod = getDuration() > 0 && (getDuration() - getCurrentTime()) <= 5;
         var canVodFailover = windowType === WindowTypes.STATIC && !aboutToEndVod;
-        var canLiveFailover = windowType !== WindowTypes.STATIC && transferFormat === 'dash';
+        var hlsSupported = windowType === WindowTypes.GROWING && liveSupport === LiveSupport.SEEKABLE;
```

Comment:
Can I be super picky and ask for this to renamed? This reads to me like it's referring to device support, rather than information on the current media. Perhaps `canHlsLiveFailover`? Could maybe extract a `canDashLiveFailover` as well.